spec/integration/orders_spec.rb

```diff
@@ -71,15 +65,24 @@
     expect(order.registry_url).not_to be_empty
   end
 
-  it 'supports create with metadata' do
+  it 'supports creation with and querying by metadata' do
```

Comment:
I feel this is the most stable strategy for now. Creating an order with metadata and then querying for it in the same spec reduces the chance of the order being on a following page